roles/openshift_health_checker/library/docker_info.py

```diff
@@ -0,0 +1,50 @@
+# pylint: disable=missing-docstring
+"""
+Ansible module for determining information about the docker host.
+"""
+
+from ansible.module_utils.docker_common import *
+
+
+class DockerInfo(DockerBaseClass):
+
+    def __init__(self, client, results):
+
+        super(DockerInfo, self).__init__()
+
+        self.client = client
+        self.results = results
+
+        self.results['info'] = self.get_info()
+
+    def fail(self, msg):
+        self.client.fail(msg)
+
+    def get_info(self):
+        """
+        Lookup and inspect information about the docker host.
+        :returns dictionary of host info.
+        """
+
+        return self.client.info()
+
+
+def main():
+    argument_spec = dict(
+        name=dict(type='list'),
+        )
+
+    client = AnsibleDockerClient(
+        argument_spec=argument_spec
+    )
+
+    results = dict(
+        changed=False,
+        info=[]
+    )
+
+    DockerInfo(client, results)
```

Comment:
> Isn't AnsibleDockerClient already exposed as some module in Ansible?
I thought this would be the place where we call docker_image_facts.

No, `docker_image_facts` [does not expose the ansible docker client](https://github.com/ansible/ansible-modules-core/blob/devel/cloud/docker/docker_image_facts.py#L235) as far as I can tell.

We do use `docker_image_facts` when we call `self.inspect_local_images`, however once we begin using skopeo after the local check, we need the `AnsibleDockerClient` in order to know which registries to target when inspecting images.